.github/actions/setup/action.yml

```diff
@@ -14,3 +14,6 @@ runs:
     - name: Install Python dependencies (uv will install python automatically, if no appropriate version is installed)
       run: uv sync --all-extras --dev
       shell: bash
+    - name: Create validate-data directory
+      run: mkdir -p /home/runner/.dsp-tools/validate-data
```

Comment:
@Nora-Olivia-Ammann This is coupled to `src/dsp_tools/commands/validate_data/constants.py`. If you modify it there, and forget to modify it here, you will run into hard-to-debug errors. I'd suggest to remove this, and instead add this to the python file:

```python
TURTLE_FILE_PATH..touch(exist_ok=True)
```